src/gemdat/trajectory.py

```diff
@@ -361,3 +361,22 @@ def filter(self, species: str | Collection[str]) -> Trajectory:
                               lattice=self.get_lattice(),
                               metadata=self.metadata,
                               time_step=self.time_step)
+
+    def split(self, n_parts: int) -> List[Trajectory]:
+        """Split the trajectory in n similar parts.
+
+        Parameters
+        ----------
+        n_parts : int
+            n_parts
+
+        Returns
+        -------
+        List[Trajectory]
+        """
+
+        interval = np.linspace(0, len(self) - 1, n_parts + 1)
+        subtrajectories = [
+            self[int(interval[i]):int(interval[i + 1])] for i in range(n_parts)
+        ]
```

Comment:
Would be nice to use pairwire here: https://docs.python.org/3/library/itertools.html#itertools.pairwise

```suggestion
        interval = np.linspace(0, len(self) - 1, n_parts + 1, dtype=int)
        subtrajectories = [
            self[start:stop] for start, stop in pairwise(interval)
        ]
```